examples/cluster/plot_dbscan.py

```diff
@@ -0,0 +1,128 @@
+"""
```

Comment:
Copyright header

Reply:
Good point. I was wondering how to deal with licensing and attribution. My best guess so far would be something like the following. Though the BSD3 license says we also need to reproduce the 3 clauses and disclaimer. But that would add excessive amount of text to the top of each file. Maybe we can deal with it in some other way? Is there prior art somewhere?
```
# BSD 3-Clause License
# 
# Copyright (c) 2007-2024 The scikit-learn developers.
# All rights reserved.
#
#
# Copyright (c) 2024, NVIDIA CORPORATION.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
```